.github/workflows/ci.yml

```diff
@@ -44,6 +44,15 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     - name: Install dependencies
+      if: matrix.python-version != '3.8'
+      run: |
+        python -m pip install uv
+        uv pip install --system --upgrade pip setuptools wheel
+        uv pip install --system --upgrade ".[all,test]"
+
+      # c.f. https://github.com/astral-sh/uv/issues/2062
+    - name: Install dependencies (Python 3.8)
+      if: matrix.python-version == '3.8'
```

Comment:
A bit of a hack, but until https://github.com/astral-sh/uv/issues/2062 gets resolves, we might as well speed up everything else.